nautobot/ipam/tables.py

```diff
@@ -621,6 +622,37 @@ class Meta(DeviceComponentTable.Meta):
         }
 
 
+#
+# IPAddress to Interface
+#
+
+
+class IPAddressToInterfaceTable(BaseTable):
+    pk = ToggleColumn()
+    id = tables.Column(linkify=True, verbose_name="ID")
+    ip_address = tables.Column(linkify=True, verbose_name="IP Address")
+    interface = tables.Column(linkify=True)
+    vm_interface = tables.Column(linkify=True, verbose_name="VM Interface")
```

Comment:
Probably should crib from something like the `CABLETERMINATION` column template so that these columns show something like `device1 > interface1` instead of just `interface1` for usability?